lmi-primitives.cc

```diff
@@ -17,5 +17,9 @@ int main() {
   double d = 123.456;
   wchar_t w = L'A';
   
+  // Added for testing double pointers.
+  double *ptr = &d;
+  double **dblptr = &ptr;
```

Comment:
I assume because of O0 they are not optimized away? But just in case (for future changes) can we use these values, i.e. print, as well?

Reply:
Thanks!